m2e-configurator/net.revelc.code.formatter/src/net/revelc/code/formatter/connector/internal/FormatterProjectConfigurator.java

```diff
@@ -125,14 +126,20 @@ private InputStream readConfigFile(Formatter formatter, ProjectConfigurationRequ
             javaConfigFile = formatter.getDefaultPath();
         }
 
-        IFile cfgFile = request.getProject().getFile(javaConfigFile);
-
-        if (!cfgFile.exists()) {
-            throw new CoreException(
-                    new Status(IStatus.CANCEL, FormatterCore.PLUGIN_ID, "Configuration file not found!"));
+        File cfgFile = new File(javaConfigFile);
+        if (!cfgFile.isAbsolute()) {
```

Comment:
this doesn't check about existence...

```new File("file.txt").isAbsolute(); // false```
```new File("file.txt").exists(); // may be true or false```

```new File("/file.txt").isAbsolute(); // true```
```new File("file.txt").exists(); // may be true or false```

so being absolute has nothing to do with existence

so the file may exist and not be absolute...  what I just don't know is if this ```request.getProject().getLocation().append()``` will be able to find the existing relative path file